src/includes/platforms/configuration/options/ignore-transactions.mdx

```diff
@@ -0,0 +1 @@
+A list of strings or regex patterns that match transaction names that shouldn't be sent to Sentry. Transactions that match these strings or regular expressions will be filtered out before they're sent to Sentry. When using strings, partial matches will be filtered out, so if you need to filter by exact match, use regex patterns instead. By default, all transactions are sent.
```

Comment:
This is (unfortunately) not true anymore. The [default for `ignoreTransactions`](https://github.com/getsentry/sentry-javascript/blob/137c5e912acb6bfcf9893ed114a3448ea68e28a1/packages/core/src/integrations/inboundfilters.ts#L8-L17) was changed to ignore health check transactions on the SDK side (in addition to the Relay filter). Afaik this only applies to JS though. 
 
```suggestion
A list of strings or regex patterns that match transaction names that shouldn't be sent to Sentry. Transactions that match these strings or regular expressions will be filtered out before they're sent to Sentry. When using strings, partial matches will be filtered out, so if you need to filter by exact match, use regex patterns instead. By default, transactions spanning typical API health check requests are filtered out.
```